flink-python/pyflink/table/tests/test_catalog_completeness.py

```diff
@@ -0,0 +1,169 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import unittest
+
+from pyflink.testing.test_case_utils import PythonAPICompletenessTestCase
+from pyflink.table import Catalog, CatalogDatabase, CatalogBaseTable, CatalogPartition, \
+    CatalogFunction, CatalogColumnStatistics, CatalogPartitionSpec, ObjectPath
+
+
+class CatalogAPICompletenessTests(PythonAPICompletenessTestCase, unittest.TestCase):
```

Comment:
I think we should add all the interface completeness test, please check the package `org.apache.flink.table.catalog`, and find out the interface we should check: 
![image](https://user-images.githubusercontent.com/22488084/59187028-f99a9400-8ba6-11e9-8f3a-563c87e7da00.png)

Reply:
Good catch. Have addressed this issue in the latest PR.